htdocs/install/mysql/tables/llx_categorie_fichinter.key.sql

```diff
@@ -0,0 +1,26 @@
+-- ============================================================================
```

Comment:
Can you rename the file llx_categorie_fichinter-fichinter so table is created only when module fichinter is enabled.